101-remove-resourcegroups/DeployThroughARM.ps1

```diff
@@ -0,0 +1,20 @@
+﻿
+#Connect to your Azure account
+Add-AzureRmAccount 
+
+#Select your subscription if you have more than one
+#Select-AzureRmSubscription -SubscriptionId "Your subcription ID here"
+
+#Create a GUID for the job
+$JobGUID = [System.Guid]::NewGuid().toString()
+
+#Set the parameter values for the template
+$Params = @{
+    "accountName" = "MyAccount2" ;
+    "regionId" = "Japan East";
+	"userName" = "Your UserName"; 
+	"password" = "Your password";
+}
+
+$TemplateURI = "https://raw.githubusercontent.com/azureautomation/automation-packs/master/101-remove-resourcegroups/azuredeploy.json"
```

Comment:
You may want to use a permalink here.  Not really a big deal, but I've noticed sometimes the URLS change which will would make this fail.  See https://help.github.com/articles/getting-permanent-links-to-files/ for getting GitHub permalink.